### ISSUE TITLE
Unable to make or receive phone calls after last update

### ISSUE THREAD

**Issue by ocrammaig** [GrapheneOS/os-issue-tracker#1464]
Hi, after the last update was installed I'm no longer able to make or receive phone calls.
When I try to make a call the screen pops up with the message 'call ended' and if someone tries to call me it goes straight to voicemail.
This is not a problem with the telephone operator (other people with the same operator have no problems). 
It would seem to be a call-only problem as messages and the internet connection work properly.

Graphene OS version: stable, TP1A.220905.004.2022090600

**Comment by wc2FiCLd** [comment]
Provided build number is beta channel according to grapheneOS website
Current newest stable is TP1A.220624.021.2022090400

**Comment by ocrammaig** [comment]
I didn't know that, but going to system update it says I'm on the stable channel and if I try to check for updates it says I'm already at the latest stable version. Should I try to change the release channel?

**Comment by at367Br549** [comment]
Same issue occurred to me today after the update (Beta Channel); the phone worked fine last night.  Can't prove causation however Consumer Cellular suggested getting a new SIM card.  After activating the card and resetting the network settings on my phone (Pixel 5a) everything worked fine.

**Comment by thestinger** [comment]
Please try resetting APN settings or changing APN settings.

**Comment by thestinger** [comment]
@ocrammaig Need information on which carrier you're using, and you were definitely using Beta if you're on TP1A.220905.004.2022090600, not stable.

**Comment by ocrammaig** [comment]
I tried resetting the APN settings and nothing changed.
My carrier is Iliad (Italy) if you need more information just ask.
Is it normal that if I change the channel to stable nothing happens and I stay in beta?

**Comment by thestinger** [comment]
> Is it normal that if I change the channel to stable nothing happens and I stay in beta?

Yes, that's completely normal. It just changes where you get updates. You can't downgrade.

**Comment by thestinger** [comment]
> My carrier is Iliad (Italy) if you need more information just ask.

Please provide the carrier config version string from the settings, device model and try to change APN configuration to see if that helps.

**Comment by Umlaut512** [comment]
Maybe you have set lte only mode and you do not have volte?

**Comment by jacekpoz** [comment]
had the same issue, fixed it by resetting APN to default and changing the preferred network type to 5G (even though I don't have 5G)

**Comment by Neoniq** [comment]
> had the same issue, fixed it by resetting APN to default and changing the preferred network type to 5G (even though I don't have 5G)

For me only "LTE only" doesn't work 😕 5g or LTE is fine
(Picked only LTE cuz someone on the forum telling it saves battery)

**Comment by r3g-5z** [comment]
Your carrier likely does not implement/support VoLTE.
